packages/viewport/src/with-viewport-match.native.js

```diff
@@ -9,6 +9,11 @@ import { mapValues } from 'lodash';
 import { createHigherOrderComponent } from '@wordpress/compose';
 import { withSelect } from '@wordpress/data';
 
+/**
+ * Internal dependencies
+ */
+import { default as viewportStore } from './store';
```

Comment:
Did I forgot how to import? 😅 
```suggestion
import viewportStore from './store';
```